src/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  color: #e5e5e5;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
this type of adding styles isn't flexible. Imagine if you have a case with 10 stars or even more?
1) use `:nth-child(-n + {{here_will_be_a_number_from_1_to_5}})`
Example:
`.my__star:nth-child(-n + 4)`
2) if your selectors have the same style use them as a group
Example:
```
.my__selector1, 
.my__selector2 {
  color: #ff0000;
}
```